<launch>
     <node pkg="planar_patch_map" type="planar_patch_map_node" >
          <remap from="/full_cloud" to="/snapshot_cloud"/>
     </node>

     <param name="target_frame" value="map"/>

     <node pkg="annotated_planar_patch_map" type="annotation2d_lifter_to_planar_patch_map_node" 
     	   name="annotation_lifter" respawn="false" output="screen">

	   <param name="target_frame" value="map"/>

	   <remap from="stereo_info" to="/stereo/stereo_info"/>
	   <param name="min_num_indist_tolerance" value="1"/>
	   <param name="max_allowed_num_outdist_tolerance" value="0"/>
	   <param name="dist_tolerance" value="-1000.0"/>
	   <param name="max_depth" value="5.0"/>
	   <param name="min_depth" value="0.1"/>
     </node>	   


       <node pkg="point_cloud_assembler" type="laser_scan_assembler_srv" output="screen"
           name="laser_scan_assembler">

           <remap from="scan_in" to="tilt_scan"/>
           <param name="tf_cache_time_secs" type="double" value="30.0" />
    	   <param name="max_scans" type="int" value="4000" />
    	   <param name="ignore_laser_skew" type="bool" value="true" />
    	   <param name="fixed_frame" type="string" value="map" />
   </node>

   <node pkg="point_cloud_assembler" type="point_cloud_snapshotter" output="screen" name="snapshotter">
    <remap from="laser_scanner_signal" to="laser_tilt_controller/laser_scanner_signal"/>
    <remap from="build_cloud" to="laser_scan_assembler/build_cloud" />
    <remap from="full_cloud" to="snapshot_cloud" />
    <param name="fixed_frame" type="string" value="map" />
   </node>

</launch>
